extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -104,6 +103,11 @@ class AmpStickyAd extends AMP.BaseElement {
       this.deferMutate(() => {
         toggle(this.element, true);
         this.viewport_.addToFixedLayer(this.element);
+        // Add border-bottom to the body to compensate space that was taken
+        // by sticky ad, so no content would be blocked by sticky ad unit.
+        const borderBottom = this.element./*OK*/offsetHeight;
```

Comment:
We already have a `padding-top` to account for the header bar displayed by the viewer. Should we use `padding-bottom` instead of `border-bottom`?